### ISSUE TITLE
Lite-xl

### ISSUE THREAD

**Issue by saimoomedits** [catppuccin/catppuccin#133]
**Are you available to help in the creation of this port? 😄 (optional)**
yes :smile: 

![lite-xl](https://user-images.githubusercontent.com/72156551/162846511-3ecd6082-4e56-4ec5-bf24-140c5dfb3fe7.png)

**Additional context 📷 (optional)**
link to the repo: [saimoomedits/lite-xl](https://github.com/saimoomedits/lite-xl) :cat:

**Comment by Spaxly** [comment]
Just tested this, looks good but some of the colors are a bit off, though its easily fixable.

**Comment by nekowinston** [comment]
Yeah that'd probably be because it's using the v0.1.3 palette, which is why this is only considered a draft now.
@saimoomedits would you be willing to update this to v0.2.0 or let someone work from your existing code?

**Comment by Redlolz** [comment]
I made some changes in a [fork](https://github.com/Redlolz/lite-xl-colors-catppuccin).
Colours have been updated, the other flavours have been added and the code is a little clearer or at least easier to update.
![lite-xl](https://user-images.githubusercontent.com/11429984/209841034-655aca70-1997-44a3-8e21-a381f74e8590.png)
